internal/sorting/sorting.go

```diff
@@ -0,0 +1,99 @@
+package sorting
+
+import (
+	"cmp"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/maruel/natural"
+
+	"github.com/hbesfb/bedfusion/internal/bed"
+)
+
+// Note that the the user will give the columns with 1-based indexing,
+// but that we convert this to zero-based indexing in .VerifyAndHandle()
+type Config struct {
+	SortType string `env:"SORT_TYPE" enum:"lex,nat" default:"lex" short:"s" help:"How the bed files should be sorted. lex = lexicographic sorting (chr: 1 < 10 < 2), nat = natural sorting (chr: 1 < 2 < 10)"`
+}
+
+// Global sorting function
+func (c Config) Sort(lines []bed.Line) ([]bed.Line, error) {
+	var sortedLines []bed.Line
+	switch c.SortType {
+	case "lex":
+		sortedLines = lexicographicSort(lines)
+	case "nat":
+		sortedLines = naturalSort(lines)
+	case "merge":
```

Comment:
I was thinking the merge sorting probably do not make sense for users to have access to, so that is why I have not provided it as an option. Or are you referring to something else here?

Reply:
no, that was exactly what I was talking about. Consider this solved ☺️